Framework/PythonInterface/plugins/algorithms/CompareSampleLogs.py

```diff
@@ -39,8 +41,8 @@ def PyInit(self):
 
         self.declareProperty(StringArrayProperty(name="SampleLogs", direction=Direction.Input, validator=validator),
                              doc="Comma separated list of sample logs to compare.")
-        self.declareProperty("Tolerance", 1e-3, validator=FloatBoundedValidator(lower=1e-7, upper=1.0),
-                             doc="Tolerance for comparison of double values.")
+        self.declareProperty("Tolerance", 1e-3, validator=FloatBoundedValidator(lower=1e-7, upper=100.0),
```

Comment:
TOFTOF chopper speeds can differ around 10. 100 is just to give some freedom. May be better would be to remove the upper limit, but I don't know any property which could have reasonable tolerance more than 100.